apps/docs/pages/builderdao-cli/solana_airdrop.md

```diff
@@ -0,0 +1,106 @@
+# The airdop command
+
+The following guide shows you how to send $KAFE and $BDR tokens to alpha testnet users.
+
+<aside>
+  🚨 Please confirm with @Daniel Gamboa before sending tokens to non-Figment
+  addresses.
+</aside>
+
+## Prerequisites
+
+- You must be an admin on Kafé.
+- You must use your Solana account registered as an admin.
+
+## Instructions
+
+1. Open the terminal on your local machine.
+2. Go to the directory where you plan to keep a local version of the Kafe repo.
+3. Clone the repository locally by running the following:
+
+   ```text
```

Comment:
Leading whitespace before the code block delimiter impacts the readability. We might want to unindent the code blocks 😄